pkgs/development/python-modules/treq/default.nix

```diff
@@ -26,20 +20,27 @@ buildPythonPackage rec {
     six
     openssl
     pyopenssl
+    tox
   ];
 
-  checkPhase = ''
-    ${pep8}/bin/pep8 --ignore=E902 treq
-    trial treq
-  '';
+  checkInputs = [
+    pep8
+    mock
+  ];
 
-  doCheck = false;
-  # Failure: twisted.web._newclient.RequestTransmissionFailed: [<twisted.python.failure.Failure OpenSSL.SSL.Error: [('SSL routines', 'ssl3_get_server_certificate', 'certificate verify failed')]>]
+  postPatch = ''
+    rm -fv src/treq/test/test_treq_integration.py
+  '';
 
   postBuild = ''
-    ${coreutils}/bin/mkdir -pv treq
-    ${coreutils}/bin/echo "${version}" | ${coreutils}/bin/tee treq/_version
-    cd docs && ${gnumake}/bin/make html && cd ..
+    # build documentation and install in $out
+    tox -e docs
+    cp -rv docs $out
```

Comment:
Should they not go in a specific folder? Note that with Python packages we typically do not bother with the docs.